pkgs/data/icons/whitesur-cursors/default.nix

```diff
@@ -0,0 +1,26 @@
+{ fetchFromGitHub, lib, stdenvNoCC }:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
If we are just copying files we can also use only fetchFromGitHub and postFetch.

Reply:
@SuperSandro2000 
I read `fetchFromGitHub` implementation and i noticed it's buggy.

When we specify `leaveDotGit = true`, it uses `fetchgit` as the fetcher and we can't specify `meta` together.
And when we don't specify `leaveDotGit`, it uses `fetchzip` as the fetcher and we can't specify `postFetch` together.

Reply:
For fetchzip you need to use extraPostFetch https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchzip/default.nix#L14 and fetchgit I am patching right now.

Reply:
@SuperSandro2000 
Thanks!
I've updated the commit.